src/pages/help-centre/_article-component.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { convertToHash } from './_utility'
+import { Header, Text } from 'components/elements'
+import { LocalizedLink, Localize } from 'components/localization'
+import device from 'themes/device'
+
+type IItems = {
```

Comment:
```suggestion
type IItemsType = {
```

 Typing convention.

Reply:
Naming this `PropsType` would be more appropriate, seeing where it is being used.